### ISSUE TITLE
Unable to obtain bootnode: invalid URL scheme, want "enode"

### ISSUE THREAD

**Issue by fab-10** [ethereum/hive#593]
At least for Besu, there are some tests that do not start and the simulator log this error:

```
FAIL (Invalid Ancestor Chain Re-Org, Invalid PrevRandao, Invalid P9', Reveal using sync): Unable to obtain bootnode: invalid URL scheme, want "enode"
```
Source: https://hivetests2.ethdevops.io/?page=v-pills-results-tab&suite=1657032249-2075290de9cb4eff7492fe284a5f5951.json

This seems to be related to the simulator asking something and Besu not always ready to respond, any idea if it is possible to tune it in the Besu client script, or it is something that needs to be changed in the test suite?

**Comment by marioevz** [comment]
@fab-10 It seems like Besu is intermittently returning `null\n` to `curl -s -X POST  -H "Content-Type: application/json"  --data {"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":1} "localhost:8545"`, which is called from this script https://github.com/ethereum/hive/blob/master/clients/besu/enode.sh, about ~1sec after besu starts.
Could it be that at this point besu is not prepared to respond ?

**Comment by fab-10** [comment]
@marioevz indeed Besu could not be always ready to respond, so I have implemented a retry mechanism in PR #595
